server.js

```diff
@@ -658,6 +645,33 @@ function setWindowTitle(title) {
 userModule.initUserStorage(dataRoot)
     .then(userModule.ensurePublicDirectoriesExist)
     .then(userModule.migrateUserData)
+    .then(async () => {
+        if (listen) {
+            if (getConfigValue('securityOverride', false)) {
+                console.warn(color.red('Security has been overridden. If it\'s not a trusted network, change the settings.'));
+                return
+            }
+            if(!enableWhitelist && !basicAuthMode && !enableAccounts) {
+                console.error(color.red('Your SillyTavern is currently unsecurely open to the public. Enable whitelisting or basic authentication or accounts authentication.'));
+                process.exit(1);
+            }
+            const users = await userModule.getAllUsers();
+            const unprotectedUsers = users.filter(x => x.admin && !x.password);
+            if (unprotectedUsers.length > 0) {
```

Comment:
I think it should only end the process if the basic auth mode is disabled. Reporting it otherwise is fine.

Reply:
fixed in [`422306c` (#2447)](https://github.com/SillyTavern/SillyTavern/pull/2447/commits/422306c2b6faed71601eb74e42f54c20294cc328)